src/widgets/mentors/ui/mentors.tsx

```diff
@@ -1,36 +1,40 @@
+import classNames from 'classnames/bind';
 import { LINKS } from '@/app/const';
 import mentorImg from '@/shared/assets/mentors-wanted.webp';
 import { Image } from '@/shared/ui/image';
 import { LinkCustom } from '@/shared/ui/link-custom';
 import { Paragraph } from '@/shared/ui/paragraph';
 import { WidgetTitle } from '@/shared/ui/widget-title';
 
-import './mentors.scss';
+import styles from './mentors.module.scss';
+
+const cx = classNames.bind(styles);
 
 export const Mentors = () => {
   return (
-    <section className="mentors container" id="mentors-wanted">
-      <div className="mentors content column-2">
-        <div className="mentors-info">
+    <section className={cx('container', 'mentors-container')} id="mentors-wanted">
+      <article className={cx('content', 'column-2', 'mentors-content')}>
+        <div className={cx('mentors-info')}>
           <WidgetTitle size="large" mods="lines">Mentors wanted!</WidgetTitle>
-          <Paragraph fontSize="large" className="paragraph">
+          <Paragraph fontSize="large" className={cx('paragraph')}>
             The Rolling Scopes School is constantly looking for mentors from all over the world to
             teach everyone who wants to learn the JavaScript language and the world of Front-end.
             Over the past few years, over 1500+ people have successfully completed our six month
             training program.
           </Paragraph>
           <LinkCustom
+            className={cx('link-custom')}
```

Comment:
due to changes in scss above, we can remove this class
```suggestion

```

Reply:
Done